components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/ApplicationConstants.java

```diff
@@ -268,10 +278,22 @@ public enum ErrorMessage {
                 "Error occurred while retrieving user by userid: %s."),
         NON_EXISTING_USER_ID("60504", "User not found",
                 "No user found for the given user-id: %s."),
+        UNSUPPORTED_USER_STORE_MANAGER(UNEXPECTED_SERVER_ERROR.getCode(), "Unsupported user store manager.",
+                "The underlying user store manager is not a unique ID user store manager for the tenant: %s."),
         ERROR_RETRIEVING_USERSTORE_MANAGER("65504", "Error retrieving userstore manager.",
-                "Error occurred while retrieving userstore manager."),
+                "Error occurred while retrieving userstore manager for the tenant: %s."),
```

Comment:
remove the renant domain change in error message

Reply:
Fixed with https://github.com/wso2/carbon-identity-framework/pull/6283/commits/53c09420810390d742076ceeb23f16b4834c679a